test-zone-data/Engine/Recursor-PM/README.md

```diff
@@ -20,6 +20,446 @@ See [CNAME.md] for specification of the scenarios.
+
+;; Query time: 0 msec
+;; SERVER: 127.30.1.31#53(127.30.1.31) (UDP)
+;; WHEN: Wed Nov 22 21:43:49 UTC 2023
+;; MSG SIZE  rcvd: 281
+```
+
+```
+; <<>> DiG 9.18.18-0ubuntu0.22.04.1-Ubuntu <<>> @127.30.1.32 looped-cname-out-of-zone.sub2.cname.recursor.engine.xa
+; (1 server found)
+;; global options: +cmd
+;; Got answer:
+;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 23787
+;; flags: qr aa rd; QUERY: 1, ANSWER: 1, AUTHORITY: 1, ADDITIONAL: 1
+;; WARNING: recursion requested but not available
+
+;; OPT PSEUDOSECTION:
+; EDNS: version: 0, flags:; udp: 1232
+; COOKIE: 5c10d02a24f656d9 (echoed)
+;; QUESTION SECTION:
+;looped-cname-out-of-zone.sub2.cname.recursor.engine.xa.	IN A
+
+;; ANSWER SECTION:
+looped-cname-out-of-zone.sub2.cname.recursor.engine.xa.	3600 IN	CNAME looped-cname-out-of-zone.sub3.cname.recursor.engine.xa.
+
+;; AUTHORITY SECTION:
+sub2.cname.recursor.engine.xa. 3600 IN	NS	ns1.sub2.cname.recursor.engine.xa.
+
+;; Query time: 0 msec
+;; SERVER: 127.30.1.32#53(127.30.1.32) (UDP)
+;; WHEN: Wed Nov 22 21:45:24 UTC 2023
+;; MSG SIZE  rcvd: 293
+```
+
+```
+; <<>> DiG 9.18.18-0ubuntu0.22.04.1-Ubuntu <<>> @127.30.1.31 looped-cname-out-of-zone.sub3.cname.recursor.engine.xa.
+; (1 server found)
+;; global options: +cmd
+;; Got answer:
+;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 36301
+;; flags: qr rd; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 3
+;; WARNING: recursion requested but not available
+
+;; OPT PSEUDOSECTION:
+; EDNS: version: 0, flags:; udp: 1232
+; COOKIE: 63d0ce8b86dec736 (echoed)
+;; QUESTION SECTION:
+;looped-cname-out-of-zone.sub3.cname.recursor.engine.xa.	IN A
+
+;; AUTHORITY SECTION:
+sub3.cname.recursor.engine.xa. 3600 IN	NS	ns1.sub3.cname.recursor.engine.xa.
+
+;; ADDITIONAL SECTION:
+ns1.sub3.cname.recursor.engine.xa. 3600	IN A	127.30.1.33
+ns1.sub3.cname.recursor.engine.xa. 3600	IN AAAA	fda1:b2:c3:0:127:30:1:33
+
+;; Query time: 0 msec
+;; SERVER: 127.30.1.31#53(127.30.1.31) (UDP)
+;; WHEN: Wed Nov 22 21:46:29 UTC 2023
+;; MSG SIZE  rcvd: 281
+```
+
+```
+; <<>> DiG 9.18.18-0ubuntu0.22.04.1-Ubuntu <<>> @127.30.1.33 looped-cname-out-of-zone.sub3.cname.recursor.engine.xa.
+; (1 server found)
+;; global options: +cmd
+;; Got answer:
+;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 51489
+;; flags: qr aa rd; QUERY: 1, ANSWER: 1, AUTHORITY: 1, ADDITIONAL: 1
+;; WARNING: recursion requested but not available
+
+;; OPT PSEUDOSECTION:
+; EDNS: version: 0, flags:; udp: 1232
+; COOKIE: 8782d3de032e92ce (echoed)
+;; QUESTION SECTION:
+;looped-cname-out-of-zone.sub3.cname.recursor.engine.xa.	IN A
+
+;; ANSWER SECTION:
+looped-cname-out-of-zone.sub3.cname.recursor.engine.xa.	3600 IN	CNAME looped-cname-out-of-zone.sub2.cname.recursor.engine.xa.
+
+;; AUTHORITY SECTION:
+sub3.cname.recursor.engine.xa. 3600 IN	NS	ns1.sub3.cname.recursor.engine.xa.
+
+;; Query time: 0 msec
+;; SERVER: 127.30.1.33#53(127.30.1.33) (UDP)
+;; WHEN: Wed Nov 22 21:47:03 UTC 2023
+;; MSG SIZE  rcvd: 293
+```
+
 To be completed.
```

Comment:
This section is now complete, isn’t it?

Reply:
I guess we could consider it to be complete. I will remove the "to be completed".